client/src/icons/icons.js

```diff
@@ -29,10 +29,12 @@ class _IconWrapper extends React.Component {
       rotation,
       title,
       width,
+      max_width,
```

Comment:
Before we get any more of these styling props, maybe the icons should just have a `svg_style` prop that takes an arbitrary style object and merges it in with the defaults in `svg_styles`? I think `vertical_align` could be something set through the `svg_style` prop. Might want to leave `height` and `width` out since they have the quirk of defaulting to a square if `height` is missing.